### PR TITLE
Fix for not showing keyboard on all devices on edittext focus (EXPOSUREAPP-4152)

### PR DESCRIPTION
**Pull request by BMItr**
This PR addresses the no-keyboard-showing-up automatically on some device when edittext has focus.

### CODE REVIEW

**Review by ralfgehrer — APPROVED** [review]
Works nicely on a Pixel and a Huawei P10.

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/util/ContactDiaryExtensions.kt [hunk 1]
```diff
@@ -40,3 +28,32 @@ fun String.formatContactDiaryNameField(maxLength: Int): String {
     }
     return newName.take(maxLength)
 }
+
+fun View.focusAndShowKeyboard() {
+    /**
+     * This is to be called when the window already has focus.
+     */
+    fun View.showTheKeyboardNow() {
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/util/ContactDiaryExtensions.kt:36]: Why is this nested?

**Review by axelherbstreith — APPROVED** [review]
lgtm

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/util/ContactDiaryExtensions.kt (hunk 1 above)
Comment (BMItter) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/util/ContactDiaryExtensions.kt:36]: member functions can be accessed without creating an object and for good overview

**Pull request merged**